Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:latest
+
+RUN yum update -y
+RUN yum install -y unzip
+LABEL ORG="Armedia LLC" \
+      APP="loki" \
+      VERSION="1.0" \
+      IMAGE_SOURCE="https://github.com/ArkCase/ark_loki" \
+      MAINTAINER="Armedia LLC"
+
+#LOKI
+ARG LOKI_VERSION="2.4.2"
+
+ENV LOKI_USERID=2000 \
+    LOKI_GROUPID=2020 \
+    LOKI_GROUPNAME=loki \
+    LOKI_USER=loki \
+    LOKI_PORT=3100 \
+    LOKI_URL="https://github.com/grafana/loki/releases/download/v${LOKI_VERSION}/loki-linux-amd64.zip"
```

Comment:
Does Loki need to be configured to listen the log stream from each other container?  Or does it automatically discover the other containers and index them automatically?  Or does this configuration go into the orchestration layer (docker-compose, helm, etc) and doesn't belong in the image itself at all?

Reply:
@david-oc-miller Loki is a Log aggregation engine.  Each log stream from the container is indexed by its labels. And _**Promtail**_ will take care of sending logs to Loki from the running containers.